tests/src/tests.rs

```diff
@@ -104,8 +132,43 @@ fn test_basic() {
 }
 
 #[test]
-fn test_destroy_udt() {
-    let (mut context, tx) = build_test_context(vec![1000], vec![800, 100, 50], false);
+fn test_ckb_sudt_all_order1() {
+    // input1: dealt_amount(50sudt 0x12A05F200u128) + undealt_amount(150sudt 0x37E11D600u128) + price(5*10^10 0xBA43B7400u64) + buy(00)
+    // input2: dealt_amount(100sudt 0x2540BE400u128) + undealt_amount(150.45sudt 0x380C07B40u128) + price(5*10^10 0xBA43B7400u64) + sell(01)
+    let inputs_data = vec![
+        Bytes::from(
+            hex::decode("00F2052A01000000000000000000000000D6117E03000000000000000000000000743BA40B00000000").unwrap(),
```

Comment:
I am trying to play around with the tests, but this string scare me away 😄 

If this string can be separated into different variables, it would be very helpful for debugging.

Reply:
Yes. I try to separate into different variables before, but I failed because of rust grammar. Maybe we can update it later.